var/spack/repos/builtin/packages/py-jprops/package.py

```diff
@@ -0,0 +1,36 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyJprops(PythonPackage):
+    """FIXME: Put a proper description of your package here."""
```

Comment:
I was trying to push the description `Java properties file parser for Python` but seems like don't have permission to update PR from your branch:

```
remote: Resolving deltas: 100% (4/4), completed with 4 local objects.
To https://github.com/matz-e/spack.git
 ! [remote rejected]     HEAD -> prepare-spykfunc (permission denied)
error: failed to push some refs to 'https://github.com/matz-e/spack.git'
```

Reply:
It's my repository… I'll pick that fix.